processing/src/main/java/org/apache/druid/segment/VectorColumnProcessorFactory.java

```diff
@@ -41,17 +41,25 @@
 public interface VectorColumnProcessorFactory<T>
 {
   /**
-   * Called when {@link ColumnCapabilities#getType()} is STRING and the underlying column always has a single value
+   * Called only if {@link ColumnCapabilities#getType()} is STRING and the underlying column always has a single value
```

Comment:
woops, I guess I only fixed up docs for `VectorColumnSelectorFactory` in #10613, should've done these too, my bad